packages/autocomplete-core/src/stateReducer.ts

```diff
@@ -55,7 +55,7 @@ export const stateReducer: Reducer = (action, state, props) => {
           1,
           state.highlightedIndex,
           getItemsCount(state),
-          props.defaultHighlightedIndex
+          action.props.defaultHighlightedIndex
```

Comment:
is there a reason these variables are nested in action.props, and not top-level in action, for an easier type signature for action?

Reply:
FYI [`defaultHighlightedIndex`](https://algolia-autocomplete.netlify.app/docs/partials/createautocomplete-props/#defaulthighlightedindex) is an Autocomplete option.

Did you mean to not retrieve this value from `props` but rather from `payload.defaultHighlightedIndex`?

Reply:
I didn't get that props meant options, it makes sense in that case then!